CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+# v0.12.1
```

Comment:
Could we add a big notice at the top here that this is a breaking change in `rekor-cli` and that users must upgrade to `rekor-cli` v0.12.1 to continue using against the public instance?

Reply:
done, thanks!